packages/network/src/composition.ts

```diff
@@ -95,15 +96,15 @@ export const createNetworkNode = ({
     location,
     trackers,
     metricsContext = new MetricsContext(id),
-    pingInterval,
+    peerPingInterval,
     disconnectionWaitTime,
     newWebrtcConnectionTimeout,
     webrtcDatachannelBufferThresholdLow,
     webrtcDatachannelBufferThresholdHigh,
     stunUrls = ['stun:stun.l.google.com:19302']
 }: NetworkNodeOptions): NetworkNode => {
     const peerInfo = PeerInfo.newNode(id, name, undefined, undefined, location)
-    const endpoint = new NodeClientWsEndpoint(peerInfo, metricsContext, pingInterval)
+    const endpoint = new NodeClientWsEndpoint(peerInfo, metricsContext, peerPingInterval)
```

Comment:
Given a node, was the intention to be able to define a separate ping interval for connection to tracker (WebSocket) and connection to other nodes (WebRTC)? If so then should this  be `trackerPingInterval`? (For clarity: the interval parameter here defines the ping-pong interval from node -> tracker).